Tools/autotest/carbonix.py

```diff
@@ -201,6 +207,73 @@ def TestGPSPrearm(index):
             # Restore everything
             self.context_pop()
 
+        def wait_for_engine_temp(condition, threshold, timeout=10):
```

Comment:
I don't like this condition-type-string thing. We should copy how wait_altitude does it, which is to pass a min and max, that lets you do all three types of comparisons with just those two more-intuitive inputs: min, max, or equal-within-a-threshold.